pkg/util/image/status_to_label.go

```diff
@@ -10,21 +10,22 @@ import (
 	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha3"
 )
 
-// SyncStatusToLabels copies the image's capabilities and OS information from
-// the image status to well-known labels on the image in order to make it
+// SyncStatusToLabels copies the image's type, capabilities and OS information
+// from the image status to well-known labels on the image in order to make it
 // easier for clients to search for images based on capabilities or OS details.
 func SyncStatusToLabels(
 	obj metav1.Object,
 	status vmopv1.VirtualMachineImageStatus) {
 
 	var (
+		imgType   = status.Type
 		osID      = status.OSInfo.ID
 		osType    = status.OSInfo.Type
 		osVersion = status.OSInfo.Version
 		caps      = status.Capabilities
 	)
 
-	if osID == "" && osType == "" && osVersion == "" && len(caps) == 0 {
+	if imgType == "" && osID == "" && osType == "" && osVersion == "" && len(caps) == 0 {
 		return
```

Comment:
Should we remove the annotations in this case? Or set their values to ""?

Reply:
Updated to remove.